### ISSUE TITLE
Text clipping to next page and Width not accepting as calc()

### ISSUE THREAD

**Issue by ronak-qp** [wkhtmltopdf/wkhtmltopdf#4481]
**wkhtmltopdf version(s) affected**: wkhtmltopdf 0.12.5 (with patched qt)

**OS information**
Fedora 28

**Description**
Text is clipping to next page, please see screenshot attached
Width not accepting as width: calc(100% / 5);, its taking by default as 100%

![ISSUE1](https://user-images.githubusercontent.com/15610387/65305992-082f1900-dba2-11e9-965b-312adc871da0.png)
![ISSUE](https://user-images.githubusercontent.com/15610387/65305993-082f1900-dba2-11e9-8f35-8d58f7e2895e.png)
![EXPECTED](https://user-images.githubusercontent.com/15610387/65305994-082f1900-dba2-11e9-8043-2e2b315c36ed.png)

**Comment by PhilterPaper** [comment]
Are these two separate problems? Your description is confusing. Regarding text "clipping to next page", are these table rows? That problem of part of a _row_ appearing at the bottom of the page and the rest of it on the next page has been reported before. I don't recall if there was any workaround for it.

**Comment by ronak-qp** [comment]
Yes these are two different problems, for the text getting clipped to next page I am not able to find any solution on any of the reported tickets. No its not table, it's a div
